src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -724,7 +724,7 @@ def get_auth_settings(cmd, resource_group_name, name, slot=None):
 
 
 def update_auth_settings(cmd, resource_group_name, name, enabled=None, action=None,  # pylint: disable=unused-argument
-                         client_id=None, token_store_enabled=None,  # pylint: disable=unused-argument
+                         client_id=None, token_store_enabled=None, runtime_version=None,  # pylint: disable=unused-argument
```

Comment:
will be param be passed to sdk call? didn't see it

Reply:
`runtime_version` is not used.

Reply:
runtime_version will be set in lines 752-4

Reply:
I am still confused. I search "runtime_version" and can't find usage.